src/app/rpmostree-builtin-cleanup.cxx

```diff
@@ -81,7 +81,15 @@ rpmostree_builtin_cleanup (int             argc,
   if (opt_rollback)
     g_ptr_array_add (cleanup_types, (char*)"rollback-deploy");
   if (opt_repomd)
-    g_ptr_array_add (cleanup_types, (char*)"repomd");
+    {
```

Comment:
Let's add some sort of comment for these special cases.  Perhaps:
```suggestion
    {
      // client-ostree-container: We need to support cleaning up metadata
```

Then we can `git grep client-ostree-container` for the likely-to-grow set of operations which can be run inside a container.

OTOH we could also `git grep rpmostreecxx::running_in_container()` I guess, so maybe ignore this and let's add the special helper API below?

Reply:
+1 cool will add the api.